cmd/influxd/server_integration_test.go

```diff
@@ -600,6 +600,7 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 		if name == "" {
 			name = tt.query
 		}
+		fmt.Printf("TEST: %d: %s\n", i, name)
```

Comment:
oh, I had it so that if there's a failure and you're tracing through, you can find that test along with whatever output was coming out of the logs then. Was very helpful for me troubleshooting something.